### ISSUE TITLE
GPU isn't used

### ISSUE THREAD

**Issue by Whisper94** [AlexeyAB/darknet#5517]
Hello. Thank you for your amazing work! 

After compiling darknet using vcpkg (another options didn't work - cmake, make...) I've tried to check demo (darknet.exe detector test cfg/coco.data cfg/yolov4.cfg yolov4.weights -thresh 0.25) 
and now I have the following issue.
![image](https://user-images.githubusercontent.com/60612672/81270439-09d32180-904b-11ea-89ea-fae0612aaa46.png)

First of all there is the message that my GPU won't be use. How can I include my GPU for recognizing?

If I'll try to train my own dataset it's look like my gpus will also not be used during training although I have two gpus nvidia 1080. VRAM doesn't increase, temperature also not. Only CPU is working with about 80°C temperature. And after some time I'll get Nan Nan Nan

![image](https://user-images.githubusercontent.com/60612672/81271147-0724fc00-904c-11ea-86d3-a12dd1e0c78a.png)

Could you explain what I made wrong? Thank you!
P.S.: I have already used different forks of yolov3 and I have installed CUDE 10.0, cuDNN, opencv4

**Comment by m1kelle-crypto** [comment]
Hello, just check this issue #5481. I think it will help you.

**Comment by AlexeyAB** [comment]
Read: https://github.com/AlexeyAB/darknet#how-to-compile-on-windows-using-vcpkg

> `.\vcpkg install darknet[full]:x64-windows`

**Comment by syg1996419** [comment]
我之前也遇到这个问题,可能是你编译的时候没有将CUDA编译成功,我把CUDA卸了重新安装10.0，cudnn7.6.5的就可以了

**Comment by Whisper94** [comment]
Thank you very much! 
I've deleted at first everything (CUDA, cDDN, opencv, VS, vcpkg ) and then I've  started clear installation https://medium.com/analytics-vidhya/installing-darknet-on-windows-462d84840e5a. Instead of using ".\build.ps1" I've used vcpkg and after 4 hours compiling it works. Now I'm train my own dataset and try to improve mAP

I have also another question. If my dataset has images that resolution is about 150x150 pixels and I'd like to detect these small objects on 4k resolution image. Is it better to train yolov4 on 32x32 input shape and then to scale 4k image to about 1088 pixels? (multiple of 32) for example as input for detection (1088x1440 pixels)?

Thank you!
